pyrsia_node/Cargo.toml

```
@@ -11,6 +11,7 @@ anyhow = "1.0.45"
 bincode = "1.3.3"
 blake3 = "1.2.0"
 bytes = "1.1.0"
+chrono = "0.4.19"
```

Lets not require old time! https://github.com/chronotope/chrono/blob/f6bd567bb677262645c1fc3131c8c1071cd77ec3/Cargo.toml#L27 

```suggestion
chrono = { verion="0.4.19", default= false, features=["clock", "std"] }
```

---

The suggested fix did not work. See https://github.com/pyrsia/pyrsia/runs/4653072658?check_suite_focus=true

I will leave this change in and put the second ignore back in.

---

@prince-chrismc I tried it again with cargo.lock updated. It still does not work.  Apparently cargo audit isn't as smart as you are.  See
https://github.com/pyrsia/pyrsia/runs/4653351944?check_suite_focus=true